### DISCUSSION TITLE
Assistance - Permission denied - /etc/nginx/nginx.conf:36

### DISCUSSION THREAD

**Discussion by JM-ET** [AykutSarac/jsoncrack.com#172] [Q&A]
Hi All.
FIrst time posting on github and please forgfive me is misposting in wrong thread. 
I've searched Issues but suspect my issue is not jsoncrack itself.

Following these steps : https://github.com/AykutSarac/jsoncrack.com#-docker

> Build a Docker image with docker build -t jsoncrack .
> Run locally with docker run -p 8888:8080 jsoncrack

Built image successfully.
But when in run it, it fails with

**➜  jsoncrack.com git:(main) docker run -p 8888:8080 jsoncrack**
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: can not modify /etc/nginx/conf.d/default.conf (read-only file system?)
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2022/09/14 09:23:46 [emerg] 1#1: open() "/etc/nginx/conf.d/default.conf" failed (13: Permission denied) in /etc/nginx/nginx.conf:36
**nginx: [emerg] open() "/etc/nginx/conf.d/default.conf" failed (13: Permission denied) in /etc/nginx/nginx.conf:36**

Does anyone know why this may occur and how to resolve?

Thanks.

**Comment by AykutSarac** [comment]
Dockerfile has been updated now.
